universal-login-contracts/contracts/Clicker.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.4.24;
+
+
+/* solium-disable security/no-block-members*/
+contract Clicker {
```

Comment:
I don't think Clicker.sol should be a part of the universal-login-contracts directory. It is a contract related to the example application and should live there. :)

Reply:
We first want the app to be working, ideally all the bits: the example, the relayer code, the sdk, should live in different repositories altogether. For now, leave it be

Reply:
I think Justyna used in tests. I think we can move it easily back to example and use mock contract for test.